docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1191,15 +1191,23 @@ async function stopScreenShare() {
   }
 }
 
+function updateLocalVideoStream(localStream) {
+  const [currLocalStream, currLocalShare] = currentMediaStreams;
```

Comment:
Where is the value for 1080p being passed , IS it from the drop down?

Reply:
Yes it will be passed from dropdown.